web/src/components/tasks/business-formation/getErrorStateForField.test.ts

```diff
@@ -109,7 +109,7 @@ describe("getErrorStateForField", () => {
           formationFormData,
           businessNameAvailability,
         }).label
-      ).toEqual(Config.formation.fields.businessName.errorInlineUnavailable);
+      ).toEqual(undefined);
```

Comment:
dust - So this is undefined, because we need to render something custom for a multi-line inline error right?

Reply:
Yes, I'll bring this up to the team at-large, per Soma's suggestion.